supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -434,6 +434,16 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*: xref:he[he]
 
+[[shebang]]
+==== image:images/yes.png[yes] shebang (noun)
+*Description*: A _shebang_ such as `#!/bin/bash` is a common term for an _interpreter directive_, which is a computer language construct that controls which interpreter parses and interprets instructions in a computer program. This term is also commonly known as "hashbang". On first mention, include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs...". Do not use "hashbang".
```

Comment:
```suggestion
*Description*: A _shebang_ is a common term for an _interpreter directive_, such as `#!/bin/bash`, which is a computer language construct that controls which interpreter parses and interprets instructions in a computer program. This term is also commonly known as "hashbang". On first mention, include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs ...". Do not use "hashbang".
```

Reply:
@mportman12 These changes aren't made. Not sure if that's deliberate or overlooked, just checking :)